### ISSUE TITLE
Error if initialize on an empty element

### ISSUE THREAD

**Issue by yellowsix** [KingSora/OverlayScrollbars#41]
If you try to initialize a scrollbar on an empty element, it gives this error:

TypeError: Argument 1 of Node.appendChild does not implement interface Node.
overlayscrollbars.js:1142:25

I can reproduce this with e.g.

```
<div id="test"></div>

OverlayScrollbars(document.getElementById("test"), { });
```

Sometimes I may wish to do this if I haven't filled that div with content yet, e.g. a dynamically loaded list. For now I have deferred initializing the scrollbar until after the first time the container has at least one thing in it, but would be nice to not have to do that.

**Comment by KingSora** [comment]
Good day!

Yes, I know this issue. Currently this is only happening if you are using the `non jQuery version` of the plugin. I've fixed this issue already though, the fix will be implemented in `version 1.4.5` which will come pretty soon.

**Comment by KingSora** [comment]
Alright, [version 1.4.5](https://github.com/KingSora/OverlayScrollbars/releases/tag/v1.4.5) is now published, and this issue should be fixed!
